src/components/accordion/accordion.spec.js

```diff
@@ -52,6 +52,9 @@ describe('Accordion', () => {
     ).mockImplementation(() => contentHeight);
   };
 
+  testStyledSystemSpacing(
+    props => <Accordion title='foo' { ...props } />, () => wrapper.find(StyledAccordionContainer)
```

Comment:
The second parameter to this function needs to be `{}` or `undefined` for the defaults, you were right the first time
```suggestion
    props => <Accordion title='foo' { ...props } />, {}, () => wrapper.find(StyledAccordionContainer)
```